Changelog.md

```diff
@@ -14,6 +14,7 @@
 
 ### Features and Improvements
 
+- Check labels on `flow.run_config` when generating warning for flow labels without an agent [#312](https://github.com/PrefectHQ/ui/pull/312)
```

Comment:
@jcrist you'll want to add this to the `Unreleased` F&I section at the top

Reply:
oops, thanks! Fixed.